bson/string_codec.go

```diff
@@ -14,7 +14,6 @@ import (
 // stringCodec is the Codec used for string values.
 type stringCodec struct {
 	// DecodeObjectIDAsHex specifies if object IDs should be decoded as their hex representation.
-	// If false, a string made from the raw object ID bytes will be used. Defaults to true.
```

Comment:
We should note that a false value will now result in a decoding error.

Reply:
I've added the comments in `DecodeContext`.